sklearn/metrics/tests/test_ranking.py

```diff
@@ -959,6 +964,51 @@ def test_precision_recall_curve_toydata():
             average_precision_score(y_true, y_score, average="micro"), 0.5
         )
 
+        y_true = np.array([[1, 0], [0, 1]])
+        y_score = np.array([[0, 1], [1, 0]])
```

Comment:
I think we can remove this, since the input is the exactly the same as the one above it.